tests/foreman/ui/test_contentview.py

```diff
@@ -2646,6 +2646,46 @@ def test_positive_add_errata_filter(session, module_org):
         )
 
 
+@tier3
+def test_positive_add_module_stream_filter(session, module_org):
+    """add module stream filter in a content view
+
+    :id: 343c543e-5773-4ea4-aff4-27e0ed6be19e
+
+    :expectedresults: content views filter created and selected module stream
+        can be added for inclusion/exclusion
+
+    :CaseLevel: Integration
+
+    :CaseImportance: High
+    """
+    filter_name = gen_string('alpha')
+    repo_name = gen_string('alpha')
+    create_sync_custom_repo(module_org.id,
```

Comment:
```suggestion
    create_sync_custom_repo(
        module_org.id, repo_name=repo_name, repo_url=CUSTOM_MODULE_STREAM_REPO_2)
```
if above doesnt sits on one line then:
```suggestion
    create_sync_custom_repo(
        module_org.id,
        repo_name=repo_name,
        repo_url=CUSTOM_MODULE_STREAM_REPO_2)
```